src/workers/compiler.ts

```diff
@@ -124,6 +124,9 @@ function virtual({
       // Compile solid code
       if (/\.(j|t)sx$/.test(filename)) {
         const babel = await loadBabel(solidVersion);
+        // HACK: Import untrack from the same solid-js version; used when logging to fake console
+        code =
+          ';import { untrack as $$untrack } from "solid-js";window.$$untrack = $$untrack;' + code;
```

Comment:
Pardon my ignorance, but why does it have to start with a `;` here?

Reply:
I initially had appended it after the code, and wanted to be safe in case it somehow got appended on the same line as existing code that did not end with a semi-colon. It's not needed anymore, I'll get rid of it.